lib/blocks/HorizontalRule.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+    // external props
+    export let props;
+    const { get } = props;
+    $: theme = props.theme;
+
+    $: data = get(theme, `data.options.blocks.${props.id}.data`, {});
+    $: border = get(data, 'border', '1px solid #cccccc');
+</script>
+
+<hr class="export-line" style="border: 0; border-bottom: {border};" />
```

Comment:
@jakubdatawrapper let us know what class/markup you would prefer for horizontal rules within the chart

Reply:
`.dw-line` please. It's already used in d3-bars and tables.